docs/ruleset_functions/get_zone_peak_internal_load_floor_area.md

```diff
@@ -0,0 +1,49 @@
+# get_zone_peak_internal_load_floor_area
+**Schema Version:** 0.0.21
+**Description:** finds the peak coincident internal loads of a zone and returns the value in btu/h/ft2
+
+**Inputs:** 
+- **RMI**
+- **zone**
+
+**Returns:**  
+- **result**: a dict giving 2 values: {"PEAK":total peak btu/h/ft2 in the zone, "AREA":total zone area} the total peak btu/sf is the internal coincident peak loads in all spaces in the zone
```

Comment:
Did we discuss whether this should be coincident or non-coincident? I don't think this is defined in 90.1 - perhaps submit an interpretation request?

Reply:
this isn't clarified in 90.1 2022 either. It simply says 'total peak internal gains'. Maybe we start with sending the Mikes and Jason a note? If this small group agrees on whether this is coincident or non-coincident then we might be fine. Doug Maddox at PNNL also implemented this for the PRM prototypes, so I'm sure this has been discussed (from which I remember this being non-coincident)

Reply:
drafted an email to the mikes.  We can discuss on the call

Reply:
also discussed that we should use design day schedules, supported by the schema as explained here - https://github.com/open229/ruleset-model-description-schema/issues/150

Reply:
Karen will request the spreadsheet Doug developed to understand how spaces within a zone need to be combined to get zone peak (average load or sum of peaks of various space loads)